cql3/Cql.g

```diff
@@ -2271,6 +2284,7 @@ K_FOR: F O R;
 K_SERVICE: S E R V I C E;
 K_LEVEL: L E V E L;
 K_LEVELS: L E V E L S;
+K_EFFECTIVE: E F F E C T I V E;
```

Comment:
I'm not familiar with the design of this feature so I wanted to ask:
Is the command "LIST EFFECTIVE SERVICE LEVEL OF"  based on some other database or something? What is the word "EFFECTIVE" supposed to refer to?

Reply:
AFAIK it's not based on anything from other databases.
We have a bunch of `LIST ...` commands already in Scylla. EFFECTIVE keyword means the command list effective service level's option of a role.

But since you are asking, that means the doc isn't that good so I've tried to make it more clear.

Reply:
Maybe I'm just being dense, but what does the word "effective" mean in the sentence "list effective service level of"? Would "list service level of ..." not be enough?
Maybe the concept of "effective service level" is something familiar and I'm just not familiar with it?

Reply:
Following the example
> There are 2 roles: role1 and role2. Role1 is assigned with sl1 (timeout = 2s, workload_type = interactive) and role2 is assigned with sl2 (timeout = 10s, workload_type = batch).

Then, if we grant role1 to role2, the user with role2 will have 2s timeout (from sl1) and batch workload type (from sl2).
So, there is no actual service level with 2s timeout and batch workload but that parameters are effectively used by users with `role2`.

So IMO "effective" keyword emphasizes that it isn't a real service level.